src/lib/common/device_info.h

```diff
@@ -13,15 +13,17 @@ const versionInfo_t *findVersionInfo(uint32_t addr, uint32_t len);
 
 bool fwIsEng(const versionInfo_t *info);
 bool fwIsDirty(const versionInfo_t *info);
-const uint32_t* getUID();
-const char * getUIDStr();
-const char * getFWVersionStr();
+const uint32_t *getUID();
+const char *getUIDStr();
+const char *getFWVersionStr();
```

Comment:
nit: as long as we're editing this, please add `void` to the parameter list

Reply:
Ok!